### DISCUSSION TITLE
Make Markdown `bold` and `italic` colorless but styled?

### DISCUSSION THREAD

**Discussion by adrian5** [2giosangmitom/nightfall.nvim#66] [General]
Going from this:
![markdown-before](https://github.com/user-attachments/assets/f7145e8f-5f7a-46c6-a004-8d345142a734)

To this:
![markdown-after](https://github.com/user-attachments/assets/2cecbb44-a400-411a-bde6-b67138119192)

I'm usually a fan of strong color separation, but I find this reads more naturally and we utilize similar colors for elements like links already. When editing, the concealed text becomes `_emphasis_` and `**boldness**` in most configurations, so it's easy enough to see.

**Comment by 2giosangmitom** [comment]
I agree with this change. Thanks for your helpful advice.
